api/api.go

```diff
@@ -42,6 +42,21 @@ func ListStates(w http.ResponseWriter, _ *http.Request, d *db.Database) {
 	}
 }
 
+// ListStatesWithLineages lists distinct State paths with associated Lineages
+func ListStatesWithLineages(w http.ResponseWriter, _ *http.Request, d *db.Database) {
+	w.Header().Set("Access-Control-Allow-Origin", "*")
```

Comment:
As discussed earlier, we might want to set this automatically on all handlers (probably not in this PR though).

Reply:
It already done on the vuejs-migration wip branch using a middleware :smile: